src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs

```diff
@@ -37,6 +38,16 @@ public ConsoleParameters(IFileHelper fileHelper)
             this.fileHelper = fileHelper;
         }
 
+#if NET451
+
+        /// <summary>
+        /// TODO: Remove the #if when project is targeted to netstandard2.0
+        /// Environment variables to be set for the process
+        /// </summary>
+        public StringDictionary EnvironmentVariables { get; set; } = new StringDictionary();
```

Comment:
>StringDictionary [](start = 15, length = 16)

new clients who take dependency on this property when they try to run with older version of TP will fail. is that a scenario?

Reply:
Yes. That is a possibility if the translation layer isn't bundled.